RubyHook.cpp

```diff
@@ -9,16 +9,40 @@
 #define ScriptName                 "RubyHook"
 #define ScriptPathParam            "script_path"
 #define SlaveRunTaskLabelDecorator "slaveRunTaskLabelDecorator"
+#define SlaveExecutorEnvironmentDecorator "slaveExecutorEnvironmentDecorator"
 #define SlaveRemoveExecutorHook    "slaveRemoveExecutorHook"
 
+namespace {
+  void hash_set(VALUE hash, const std::string& key, const std::string& value) {
+    rb_hash_aset(hash, rb_str_new_cstr(key.c_str()), rb_str_new_cstr(value.c_str()));
+  }
+
+  void hash_set(VALUE hash, const std::string& key, VALUE value) {
+    rb_hash_aset(hash, rb_str_new_cstr(key.c_str()), value);
+  }
+
+  bool hashNotNullOrEmpty(VALUE hash) {
```

Comment:
name is overkill makes me think (bad!): how about isNonEmptyHash ?

Reply:
done